time.c

```diff
@@ -5242,6 +5242,27 @@ time_strftime(VALUE time, VALUE format)
     }
 }
 
+/*
+ *  call-seq:
+ *    xmlschema(fraction_digits=0) -> string
+ *
+ *  Returns a string which represents the time as a dateTime defined by XML
+ *  Schema:
+ *
+ *    CCYY-MM-DDThh:mm:ssTZD
+ *    CCYY-MM-DDThh:mm:ss.sssTZD
+ *
+ *  where TZD is Z or [+-]hh:mm.
+ *
+ *  If self is a UTC time, Z is used as TZD.  [+-]hh:mm is used otherwise.
+ *
+ *  +fraction_digits+ specifies a number of digits to use for fractional
+ *  seconds.  Its default value is 0.
+ *
+ *      t = Time.now
+ *      t.iso8601  # => "2011-10-05T22:26:12-04:00"
```

Comment:
```suggestion
 *      t.xmlschema  # => "2011-10-05T22:26:12-04:00"
```

We may want to mention that this is aliased as `iso8601`.

Reply:
Yeah, right, I just copy-pasted what `time.rb` did :see_no_evil: